tests/storage-spec.js

```diff
@@ -155,7 +163,7 @@ describe('languageStorage factory', function () {
                     label: 'Polski',
                 },
                 {
-                    id: 'en_us',
+                    id: 'en',
```

Comment:
I'd say it's a nice improvement but since we don't need that atm, could we ignore it for now and ship this version, so it can be put in the project?